pipeline/metadata/6.0.yaml

```diff
@@ -901,6 +901,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rgw
+        - name: "test-rgw-ms-dbr-disable"
+          execution_time: "1h 37m 31s"
+          suite: "suites/quincy/rgw/tier-2-_rgw_ms-dbr-disable.yaml"
```

Comment:
@ckulal , could you fix the typo in the suite file name

Reply:
done